pages/api/program/[id].ts

```diff
@@ -35,6 +37,32 @@ export default async function handle(
         // Delete program with provided programId
         const program = await deleteProgram(id as string);
 
+        if (!program) {
+            ResponseUtil.returnNotFound(
+                res,
+                `Program with id ${id} not found.`,
+            );
+            return;
+        }
+        ResponseUtil.returnOK(res, program);
+        return;
+    } else if (req.method == "PUT") {
+        // validate new body
+        const validationError = validateCreateProgram(
```

Comment:
If the validation is same for create and update, let's rename the function name

Reply:
Fixed.